### PR TITLE
VTCCA-4593 update request a dvr confirmation page

### PR DESCRIPTION
**Pull request by michjanowicz**
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4593

### CODE REVIEW

File: app/views/includes/assessmentHeading.scala.html [hunk 1]
```diff
@@ -6,12 +6,12 @@
 <div id="assessment-heading">
     <ul id="assessment-heading" class="govuk-list">
         <li id="local-authority-reference">
-            <span class="govuk-body-s govuk-!-font-weight-bold">@messages("assessments.localAuthorityRef")</span>
-            <span class="govuk-body-s" id="local-authority-reference-code">@localAuthorityReference</span>
+            <span class="govuk-body-m govuk-!-font-weight-bold">@messages("assessments.localAuthorityRef")</span>
```
Comment (darrell-plant) [inline app/views/includes/assessmentHeading.scala.html:9]: This change is outside the scope of the Jira and will affect all pages. Are you bringing `assessmentHeading` inline with other services here?

**Comment by darrell-plant** [comment]
LGTDz 👍

File: app/views/includes/assessmentHeading.scala.html (hunk 1 above)
Comment (michjanowicz) [reply app/views/includes/assessmentHeading.scala.html:9]: I made font size bigger, following prototype. I know is affecting all the views in PL, however, recently  I did https://jira.tools.tax.service.gov.uk/browse/VTCCA-4471, where the task was to increase font size across the whole check journey.

File: app/views/dvr/requestedDetailedValuation.scala.html [hunk 2]
```diff
@@ -16,25 +16,55 @@
...
-@(submissionId: String, address: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(submissionId: String, address: String,  localAuthorityRef: String, clientDetails: Option[ClientDetails] = None)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
-@mainLayout(pageTitle=Messages("detailedValuationRequested.page.title")) {
+
+@managePropertiesUrl = @{
+    clientDetails match {
+        case Some(client) =>
+            config.dashboardUrl(
+                s"selected-client-properties?clientOrganisationId=${client.organisationId}&clientName=${client.organisationName}")
```
Comment (PeterPerhac) [inline app/views/dvr/requestedDetailedValuation.scala.html:30]: string interpolation like this isn't safe for URLs but I think we do this in many other places. I think it will be good to have a test of what happens when a client org name is "foo&clientName=bar" and review the current situation in codebase, just how widely used is string interpolation and fix it - URL-encoded strings or use the `uri-template` library (out of scope for this ticket)

**Comment by PeterPerhac** [comment]
👍  looks ok to me
